SLAVES.md

```diff
@@ -27,6 +27,17 @@ Two special NPM packages are also required for Medic builds on iOS: `ios-sim` an
 
     npm install -g ios-deploy ios-sim
 
+To run Appium tests on real iOS devices you may need to install ios-webkit-debug-proxy using the command:
+
+ ``` center
+ # The first command is only required if you don't have brew installed.
+ > ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
Please point the docs to the official Homebrew installation page, since this command might change.